html-forms/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+
+  <title>html-forms</title>
+</head>
+<body>
+  <h1>Coding Form!</h1>
+
+  <div class="form-example">
+    <label for="email">Email: </label>
+    <div><input type="text" name="name" id="name" required></div>
+  </div>
+
+    <label for="email">Why is coding cool?: </label>
+    <div><select name="devices">
+      <option value="Select an option">Select an option</option>
+      <option value="Because it's interesting">Because it's interesting</option>
+      <option value="Because it's fun">Because it's fun</option>
+      <option value="Because it's powerful">Because it's powerful</option>
+      </select>
+    </div>
+    <div>Favoriate Language (choose one)
+      <div><input type="radio" name="type" value="HTML" /> HTML
+      <input type="radio" name="type" value="CSS" />
+      CSS
+      <input type="radio" name="type" value="Javascript" />
+      Javascript</div>
+    </div>
+
+  <div>Would you like to sign up for more information?</div>
```

Comment:
Use label here, you can still wrap it in a div but the labeling text for an input should be inside a label element.